[2024-06-18T15:38:37.886+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T15:38:37.902+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task manual__2024-06-18T10:07:35.050682+00:00 [queued]>
[2024-06-18T15:38:37.907+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task manual__2024-06-18T10:07:35.050682+00:00 [queued]>
[2024-06-18T15:38:37.908+0530] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-18T15:38:37.924+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-18 10:07:35.050682+00:00
[2024-06-18T15:38:37.927+0530] {standard_task_runner.py:63} INFO - Started process 35748 to run task
[2024-06-18T15:38:37.930+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'manual__2024-06-18T10:07:35.050682+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpmx6k8sw7']
[2024-06-18T15:38:37.931+0530] {standard_task_runner.py:91} INFO - Job 18: Subtask backup_task
[2024-06-18T15:38:37.974+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task manual__2024-06-18T10:07:35.050682+00:00 [running]> on host FN-21052948.
[2024-06-18T15:38:38.030+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T10:07:35.050682+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-18T10:07:35.050682+00:00'
[2024-06-18T15:38:38.030+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T15:38:38.039+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T15:38:38.040+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-18T15:38:38.046+0530] {subprocess.py:86} INFO - Output:
[2024-06-18T15:38:38.047+0530] {subprocess.py:93} INFO - /usr/bin/bash: /home/kiit/airflow/bashScripts/backup.sh: Permission denied
[2024-06-18T15:38:38.047+0530] {subprocess.py:97} INFO - Command exited with return code 126
[2024-06-18T15:38:38.048+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T15:38:38.055+0530] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/kiit/airflow-env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/kiit/airflow-env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/kiit/airflow-env/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/kiit/airflow-env/lib/python3.8/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 126.
[2024-06-18T15:38:38.057+0530] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=Backup_Database, task_id=backup_task, run_id=manual__2024-06-18T10:07:35.050682+00:00, execution_date=20240618T100735, start_date=20240618T100837, end_date=20240618T100838
[2024-06-18T15:38:38.071+0530] {standard_task_runner.py:110} ERROR - Failed to execute job 18 for task backup_task (Bash command failed. The command returned a non-zero exit code 126.; 35748)
[2024-06-18T15:38:38.103+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-18T15:38:38.124+0530] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T15:38:38.134+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
